README.md

```diff
@@ -82,6 +82,13 @@ To **disable** the responsiveness either add the `no-responsive` attribute or ch
 <a ob-lazytube no-responsive href="http://www.youtube.com/watch?v=J---aiyznGQ">Keyboard cat</a>
 ```
 
+### Custom placeholder image (instead of Youtube autogen. thumbnail)
+
+Use your own placeholder image from your server.
+
+```html
+<a ob-lazytube href="http://www.youtube.com/watch?v=J---aiyznGQ" ob-placeholder="{{'images/myVideo.png'}}">Keyboard cat</a>
```

Comment:
Yes, if you want to use a variable/expression then you need the `{{}}`, but in the README example it's a static string – then you won't need it.